src/js/themes/base.js

```diff
@@ -912,6 +912,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     grommet: {
       // extend: undefined
     },
+    header: {
+      sticky: {
+        zIndex: '1000',
```

Comment:
Can we add a comment for why this value is what it is and how we arrived at the value? 1000 seems a bit excessive.

Reference zIndex of Drop and Layer.

Reply:
yes It is a little excessive Drop and Layer are both 20 in the theme so I can change this to 50. I wanted to test out Global Header with this and see how that would layout